### PR TITLE
[MIG] default_warehouse_from_sale_team: Migration to 17.0 T#79454

### PR DESCRIPTION
**Pull request by xmglord**
- Adapt the use of name_get as part of [1].
- Change qty_done to quantity and picked = True as part of [2].

[1] odoo/odoo#122085
[2] odoo/odoo#137864

### CODE REVIEW

**Comment by xmglord** [comment]
@luisg123v please review.

File: default_warehouse_from_sale_team/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    section_id = fields.Many2one("crm.team", string="Sales Team")
+    """Avoid access errors when user has access to a journal item but not to its journal, e.g. when
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/models/account_journal.py]: Docstring doesn't apply here, it should be a standard comment.

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by luisg123v** [comment]
You forgot to fill PR's description.

**Comment by luisg123v** [comment]
> You forgot to fill PR's description.

Ignored :(

I just did it by you.

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Pull request merged**